firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/FunctionCallingConfig.kt

```diff
@@ -44,4 +50,24 @@ class FunctionCallingConfig(val mode: Mode, val allowedFunctionNames: List<Strin
      */
     NONE
   }
+
+  companion object {
+    /**
+     * The default behavior for function calling. The model calls functions to answer queries at its
+     * discretion
+     */
+    @JvmStatic fun auto() = FunctionCallingConfig(Mode.AUTO)
+
+    /** The model always predicts a provided function call to answer every query. */
+    @JvmStatic
+    @JvmOverloads
+    fun any(allowedFunctionNames: List<String>? = null) =
```

Comment:
Is `any` the right name for always using a function call? We previously used `always`.

Reply:
it's the one used in the backend and other platforms, so we are aligning with them